changelog/4112.improvement.rst

```diff
@@ -0,0 +1 @@
+Passing pytest.mark.usefixtures to pytest.param now raises ValueError instead of silently doing nothing.
```

Comment:
I suggest we tweak this a bit:

```suggestion
Using :ref:`pytest.mark.usefixtures <pytest.mark.usefixtures ref>` on :func:`pytest.param` now produces an error instead of silently doing nothing.
```